<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="welcome_lbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome User!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_but">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>127</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>680</width>
     <height>680</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>product_name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>product_price</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="add_but">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>80</y>
     <width>127</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_but">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>130</y>
     <width>127</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_but">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>180</y>
     <width>127</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sell_but">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>230</y>
     <width>127</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>Sell</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finish_but">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>280</y>
     <width>127</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lst">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>350</y>
     <width>291</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
